datafusion/wasmtest/src/lib.rs

```diff
@@ -185,26 +206,56 @@ mod test {
 
     #[wasm_bindgen_test(unsupported = tokio::test)]
     async fn test_parquet_write() {
```

Comment:
It seems to me that this test is now redundant with "test_read_and_write" but looks good to me